doc/devdocs/readme.md

```diff
@@ -79,18 +91,20 @@ modify --installpath "%ProgramFiles(x86)%\Microsoft Visual Studio\2019\%targetFo
 
 - Open `powertoys.sln` in Visual Studio, in the `Solutions Configuration` drop-down menu select `Release` or `Debug`, from the `Build` menu choose `Build Solution`.
 - The PowerToys binaries will be in your repo under `x64\Release`.
-- If you want to copy the `PowerToys.exe` binary to a different location, you'll also need to copy the `modules` and the `svgs` folders.
 
 ## Building the Installers
 
 Our installer is two parts, an EXE and an MSI.  The EXE contains the MSI and handles more complex install logic. 
 - The EXE installs all prerequisites and installs PowerToys via the MSI. Also has additional features, such as silent installation flags
 - The MSI installs PowerToys.
 
+The installers support only the `Release` configuration, building the `Debug` configuration is not supported.
+
 ### Prerequisites Building the Installer (.MSI)
 
-1. Install the [WiX Toolset Visual Studio 2019 Extension](https://marketplace.visualstudio.com/items?itemName=RobMensching.WiXToolset).
-2. Install the [WiX Toolset build tools](https://wixtoolset.org/releases/).
+1. Before building the installers, you need to build `tools\BugReportTool\BugReportTool.sln`: in Visual Studio, in the `Solutions Configuration` drop-down menu select `Release`, from the `Build` menu choose `Build Solution`.
```

Comment:
this looks more like it belongs at line 116, it isn't a prereq for being able to compile

Reply:
If you don't build the BugReportTool solution, the MSI build will fail.

Reply:
Looking at the larger doc, we need to redo the installer section

I misread the line 116 based on the limited viewport of the diff, we need to expressly state things that need to be compiled in release mode before doing the setup / bootstrapper.

## Compile Installer

Compiling in `release mode`, step 1 and 2 must be done before the MSI will be able to be compiled.

1. Compile PowerToys.sln.  Instructions are listed above.
2. Compile Bug reporting tool.  Path from root: `PATH`
3. Compile PowerToysSetup.sln Path from root: `PATH`

Reply:
i'm happy to handle this in a different PR but i do think this needs some adjustment

Reply:
I made some changes https://github.com/microsoft/PowerToys/blob/dev/enricogior/dev-docs-update/doc/devdocs/readme.md#compile-the-installer